Tchap/Config/BuildSettings.swift

```diff
@@ -252,7 +252,7 @@ final class BuildSettings: NSObject {
     static let allowInviteExernalUsers: Bool = true
     
     // MARK: - Side Menu
-    static let enableSideMenu: Bool = true
+    static let enableSideMenu: Bool = true && !newAppLayoutEnabled
```

Comment:
Should we have this update in element ios ?

Reply:
This code is from Element iOS.